src/localization/en.js

```diff
@@ -116,15 +116,27 @@ export default {
       predictions:     'Predictions',
     },
 
+    formHelpers: {
+      type:    ' Type',
```

Comment:
That's a good point, and definitely not what I was considering. That change would take a bit of a deeper dive into the code, though. Maybe we pull this and you create an issue for that @ethanbb, that one of us gets to right quick?

Also, do we create a component for 'Income Type' and 'Expense Type'? Or does the form section component set that?

Reply:
I was just suggesting passing the whole string "Income Type", "Expense Type".... in as a property (through the snippets on each page) rather than just "Income" or "Expense." We wouldn't pass in the whole header component.

I can make an issue for it after this is merged. It's not an issue for this PR, just something related that I noticed.

Reply:
I'm not saying we'd make a new component on each page. I was wondering if we would make two components, both in 'formHelpers.js' for example, that could be used repeatedly, in case we change something important about the 'Income Type' and 'Expense Type' so that it won't have to be changed in multiple snippets.

Reply:
OK I see. What if instead of doing that, we just put those strings in a different section of the snippets, maybe just under `visitPage`, and pull them from there on all the pages that need them? That way we don't have to make any special functionality within the component for those strings, but we're still not repeating ourselves.

Something like
```
visitPage: {
  incomeHeader: "Income Type",
  expenseHeader: "Expense Type",

  stepBar: {
...
```

Or we could also put them under a key like 'common' or 'shared' if we want to keep everything at the same level of nesting.

Reply:
How would the form sections get access to those properties? They only have the snippets for themselves, they don't have the `VisitPage` snippets. I'm not sure I want to hand down another property to each form section. It might be most simple, in the end, to make a component for those two things.

Reply:
Oh good point. So to be clear, when you say "make a component for" Income type and expense type, do you mean specialized versions of `IntervalColumnHeader` with those headings plugged in, or do you mean components that *just* render the heading text? (The first one makes somewhat more sense to me.)

Reply:
I'm not really attached to an implementation right now. I can see how specialized versions could be clearer to read elsewhere, though it'd be good to do it in a way that avoids duplicate code.

Reply:
So, that said, should we make an issue for the implementation of that and just leave this snippet as is for now?

Reply:
Well, let's get rid of this snippet for now, since we're not planning to use it after we make that change (we'll either use snippets on individual pages or 'Income Type'/'Expense Type' snippets that will be added). So @ValerieKenyon can you get rid of this line with `... ' Type'` please?